Formula/xclogparser.rb

```diff
@@ -0,0 +1,17 @@
+class Xclogparser < Formula
+  desc "Tool to parse the SLF serialization format used by Xcode"
+  homepage "https://github.com/spotify/XCLogParser"
+  url "https://github.com/spotify/XCLogParser/archive/0.2.8.tar.gz"
+  sha256 "d8368a561c8e3d4a7ce049feaa5f6bc6fbda532c8c0cc8bc44feab61c9aab0f9"
+
+  depends_on :xcode
```

Comment:
```suggestion
  depends_on :xcode => ["11.0", :build]
```

(assuming it can run at runtime with Xcode deleted)